### PR TITLE
Add methodology page to index, minor cleanup

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb [hunk 1]
```diff
@@ -5,23 +5,20 @@
...
-    "|  $S^{+-}_{t,\\lambda}$   |  corrected sample data of +- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |\n",
-    "|  $S^{-+}_{t,\\lambda}$   |  corrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |\n",
-    "|  $S^{--}_{t,\\lambda}$   |  corrected sample data of -- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction    |\n",
+    "|  $I^{\\pm\\pm}_{t,\\lambda}$   |  uncorrected sample data of ++/+-/-+/-- channels |  data arrays with columns for $I$, $\\lambda$, $t$   |  neutron data |\n",
+    "|  $S^{\\pm\\pm}_{t,\\lambda}$   |  corrected sample data of ++/+-/-+/-- channels  |  data arrays with columns for $I$, $\\lambda$, $t$   | by pol-correction  |\n",
     "|  $p^{cell}$   |  pressure of <sup>3</sup>He-polarizer/analyzer  |  scalar, 1-dim   | from metadata  |\n",
     "|  $l^{cell}$   |  length of <sup>3</sup>He-polarizer/analyzer   |  scalar, 1-dim   | from metadata   |\n",
     "|  $T^{g, cell}(\\lambda)$   |  transmission of empty glass of polarizer/analyzer   |  scalar, 1-dim   | from metadata  |\n",
```
Comment (nvaytet) [inline docs/user-guide/sans-polarization-analysis-methodology.ipynb:24]: If we want to be really picky, I think the commonly accepted convention is that anything in italics is a variable (i.e. something that can have different values), and any other sub/super-scripts should be in normal font.
So I guess `g` should not be in italics here as it codes for "glass"?

Same for `DB`, `3He`, etc. below.

But it's not that important, as I said, just being picky.

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb [hunk 2]
```diff
@@ -237,7 +245,8 @@
    "id": "999b0254-3f3f-401b-8b51-424fc4971407",
    "metadata": {},
    "source": [
-    "### Literature: \n",
+    "## Literature\n",
+    "\n",
     "1) Wai Tung Lee et al., Polarisation Development at the European Spallation Source\n",
     "2) Kathryn Krycka et al., Polarization-analyzed small-angle neutron scattering. I. Polarized data reduction using Pol-Corr\n",
     "3) A. Wildes, The polarizer-analyzer correction problem in neutron polarization analysis experiments\n",
```
Comment (nvaytet) [inline docs/user-guide/sans-polarization-analysis-methodology.ipynb:252]: Be consistent with either first names or initials?

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb [hunk 3]
```diff
@@ -256,16 +265,15 @@
     "\n",
     "3: SKADI:<br>\n",
     "operated with a movable supermirror V-cavity polarizer, a spin flipper, and a movable ex-situ SEOP/MEOP <sup>3</sup>He-analyzer:<br>\n",
-    "--> The supeprmirror polarizer (time-independent and wavelength-dependent) and spin flipper transmission values can be pre-characterized and inserted to the workflow as constant parameters. For the analyzer, the current workflow can be adapted.\n",
-    "\n"
+    "--> The supeprmirror polarizer (time-independent and wavelength-dependent) and spin flipper transmission values can be pre-characterized and inserted to the workflow as constant parameters. For the analyzer, the current workflow can be adapted.\n"
```
Comment (nvaytet) [inline docs/user-guide/sans-polarization-analysis-methodology.ipynb:268]:
```suggestion
    "--> The supermirror polarizer (time-independent and wavelength-dependent) and spin flipper transmission values can be pre-characterized and inserted to the workflow as constant parameters. For the analyzer, the current workflow can be adapted.\n"
```

**Review by nvaytet — APPROVED** [review]
Just some minor comments.

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb (hunk 1 above)
Comment (SimonHeybrock) [reply docs/user-guide/sans-polarization-analysis-methodology.ipynb:24]: There are many instances of this, and yes, I think it should be fixed, but it is unrelated to this PR.

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb (hunk 2 above)
Comment (SimonHeybrock) [reply docs/user-guide/sans-polarization-analysis-methodology.ipynb:252]: Unrelated?

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb (hunk 1 above)
Comment (astellhorn) [reply docs/user-guide/sans-polarization-analysis-methodology.ipynb:24]: As this is your convention, of course yes! i will change it in the other PR / updated workflows I am writing for the instrument-specific details (e.g. where some data has to be acquired in a different way due to non-movable cells, samples, etc., and also the usage of supermirrors).

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb (hunk 2 above)
Comment (astellhorn) [reply docs/user-guide/sans-polarization-analysis-methodology.ipynb:252]: Again for me to be changed in other PR

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb (hunk 1 above)
Comment (SimonHeybrock) [reply docs/user-guide/sans-polarization-analysis-methodology.ipynb:24]: > As this is your convention

I think this is pretty much standard in all scientific literature, right, i.e., not our standard?

**Review by astellhorn — COMMENTED** [review]
Anyhow, this is merged already. But what will be about the changes on the documentary regarding sigma_0 I have made last week? We think that no errorbar should be added.

**Comment by SimonHeybrock** [comment]
> Anyhow, this is merged already. But what will be about the changes on the documentary regarding sigma_0 I have made last week? We think that no errorbar should be added.

Right, unfortunately GitHub does not manage to resolve the merge conflict automatically, so you will have to do it by hand (or copy your changes to the moved document).

> We think that no errorbar should be added.

Shouldn't we add least add one that reflects the number of digits we know?

File: docs/user-guide/sans-polarization-analysis-methodology.ipynb (hunk 1 above)
Comment (astellhorn) [reply docs/user-guide/sans-polarization-analysis-methodology.ipynb:24]: You are right

**Comment by astellhorn** [comment]
> Right, unfortunately GitHub does not manage to resolve the merge conflict automatically, so you will have to do it by hand (or copy your changes to the moved document).

So I again add all changes I made last week to the renamed document in the current main?

**Comment by astellhorn** [comment]
> Shouldn't we add least add one that reflects the number of digits we know?

Hm, that might confuse readers? Cause then theyd expect we have this errorbar measured or some more deeper reason to apply it?

**Comment by SimonHeybrock** [comment]
> > Shouldn't we add least add one that reflects the number of digits we know?
> 
> Hm, that might confuse readers? Cause then theyd expect we have this errorbar measured or some more deeper reason to apply it?

Isn't it more important to get the correct error bars on the final result? We should not just ignore sources of uncertainty, I suppose?

**Comment by SimonHeybrock** [comment]
> > Right, unfortunately GitHub does not manage to resolve the merge conflict automatically, so you will have to do it by hand (or copy your changes to the moved document).
> 
> So I again add all changes I made last week to the renamed document in the current main?

Yes, sorry about that, I did not notice that there were concurrent changes to the same file. Is it many changes?

**Comment by astellhorn** [comment]
> Isn't it more important to get the correct error bars on the final result? We should not just ignore sources of uncertainty, I suppose?

Sure, but it will be the lowest contributing part of the whole workflow - and as I do not find a listed error bar for this exact value, I am unsure to which preciseness it is measured. It is fine with me to just add an artificial errorbar according to the unit we are referring to, i.e., plus/minus 1 barn if the value is 2966 barn.

**Pull request merged**